src/containers/other/CustomHead/CustomHead.tsx

```diff
@@ -0,0 +1,53 @@
+//-React Import
+import { useEffect, useRef, useState } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { useLocation } from 'react-router-dom';
+
+//- Library Imports
+import { zNAFromPathname } from 'lib/utils';
+
+//- Utils Imports
+import { getHeadData, HeadDataType } from './CustomHead.utils';
+
+const CustomHead = () => {
+	///////////
+	// Data  //
+	///////////
+	const isMounted = useRef<boolean>();
+	const { pathname } = useLocation();
+	const zna = zNAFromPathname(pathname);
+	const [data, setData] = useState<HeadDataType>();
+
+	/////////////
+	// Effects //
+	/////////////
+	useEffect(() => {
+		isMounted.current = true;
+		const headData = getHeadData(zna);
+		setData(headData);
+		return () => {
+			isMounted.current = false;
+		};
+	}, [zna]);
+
+	return (
+		<Helmet>
```

Comment:
If this turns out to be the chosen approach to handle Head Data, something I wanted to point out here is that we are only setting values for title, favicon16x16 and favicon32x32. We should really include the full favicon package as seen in ```index.html``` for the Zero icons.

Reply:
Awesome - I've had a play around with `Helmet` before and it's a handy lil' library. Good choice of tooling!